site/docs/tutorial/ios-app.md

```diff
@@ -147,6 +160,10 @@ following load statement to the beginning of your `BUILD` file:
 load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application")
 ```
 
+We only need to load the `ios_application` rule because currently the 
+`objc_library` rule is built into the bazel package. Eventually the
```

Comment:
Let's not add statements about what will happen in the future of Bazel development in official docs. Official docs should only reflect the current state of things and not make any promises about future functionality.

Reply:
Fixed